pubsub/azure/eventhubs/metadata.yaml

```diff
@@ -97,7 +97,7 @@ metadata:
     description: |
       Storage container name.
     example: '"myeventhubstoragecontainer"'
-  - name: consumerId
+  - name: consumerID
```

Comment:
All our fields are case-insensitive. If you want to update the `metadata.yaml` and the docs, that's all that needs to change :)

Reply:
And additionally, keep in mind that `metadata.yaml` is not what controls the accepted values - this is just documentation.

Our Kit/Decode Metadata function which is based on mitchellh/mapstructure uses the WeakDecode function which accepts any casing. There are several other convenience features, which is why all metadata should always be specified as struct tags and then decoded using this special function.

Also FYI, `consumerID` is a special metadata which actually is **NOT** defined in `components-contrib`. This is hardcoded in Dapr runtime, so you might need to verify in that repo what casing is accepted. Unlike in contrib, they don't use our metadata handling code in runtime.

Reply:
ahh, okay. Docs do reflect the changes I'm making here, so that works out great and will require no doc updates: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-azure-eventhubs/